openreview/matching.py

```diff
@@ -4,67 +4,67 @@
 
 def get_conflicts(author_profiles, user_profile):
 
-    authors_domain_conflicts = set()
-    authors_email_conflicts = set()
-    authors_relation_conflicts = set()
+    author_domains = set()
+    author_emails = set()
+    author_relations = set()
 
     for author_email, profile in author_profiles.iteritems():
 
-        author_conflicts = get_author_conflicts(profile, author_email)
+        author_info = get_author_info(profile, author_email)
 
-        authors_domain_conflicts.update(author_conflicts['domains'])
-        authors_email_conflicts.update(author_conflicts['emails'])
-        authors_relation_conflicts.update(author_conflicts['relations'])
+        author_domains.update(author_info['domains'])
+        author_emails.update(author_info['emails'])
+        author_relations.update(author_info['relations'])
 
-    user_conflicts = get_profile_conflicts(user_profile)
+    user_info = get_profile_info(user_profile)
 
     conflicts = set()
-    conflicts.update(authors_domain_conflicts.intersection(user_conflicts['domains']))
-    conflicts.update(authors_relation_conflicts.intersection(user_conflicts['emails']))
-    conflicts.update(authors_email_conflicts.intersection(user_conflicts['relations']))
+    conflicts.update(author_domains.intersection(user_info['domains']))
+    conflicts.update(author_relations.intersection(user_info['emails']))
+    conflicts.update(author_emails.intersection(user_info['relations']))
+    conflicts.update(author_emails.intersection(user_info['emails']))
```

Comment:
why are you doing this? this is the same that domains conflicts

Reply:
ah, you're right that in practice this will be the same thing.

I was trying to catch the case where the reviewer is an author, but if the reviewer is an author, then there is a domain conflict. i'll remove it.